library/src/scss/components/_siteNav.scss

```diff
@@ -146,8 +145,11 @@ $siteNav-node_padding               : $utility-baseUnit - $siteNav-node_borderWi
 
 
 .siteNavNode-buttonOffset {
+    position: relative;
     display: flex;
     justify-content: flex-end;
     width: $siteNav-nodeToggle_width;
     margin-left: -#{$siteNav-nodeToggle_width};
+    top: #{(37 / 2)}px;
```

Comment:
Seem like hardcoded value (37). Could we base it on some global variable value? do we have default `line-height` or `siteNav-button-height` or `siteNav-height` ?

Reply:
I'll fix it when I convert the styles to JS.

Reply:
Why not now? Never mind